### PR TITLE
Add missing install-chef-suse variables

### PR DESCRIPTION
**Pull request by rsalevsky**
These variables got accidentally removed by ae3555f.

Superseeds #2181 and #2180.

### CODE REVIEW

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -292,6 +292,10 @@ EOF
     exit 1
 fi
 
+IPv4_addr=$( echo "$resolved" | awk '{ if ($1 !~ /:/) { print $1; exit } }' )
```
Comment (rhafer) [inline scripts/install-chef-suse.sh]: ```$resolved``` still does not exist anywhere.

**Comment by tboerger** [comment]
LGTM, gating fails because of tgt service (so it seems to be unrelated).

**Comment by rhafer** [comment]
+1

**Comment by jsuchome** [comment]
+1, but we have to remove current (reverting) patch from the package

**Comment by rhafer** [comment]
@jsuchome I'll do that as soon as this one is merged.

**Pull request merged**